Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -188,6 +207,17 @@ public void driveGrabber(double position) {
      * @param targetPosition target position for slides motors in ticks
      */
     public void driveSlides(int targetPosition) {
+        if(targetPosition == 0) {
+            if(limitSwitch.getState()) {
```

Comment:
Can you add a comment explaining what `limitSwitch.getState()` does?

Reply:
Added comments to all the code

Reply:
Thanks! I'm glad that doing the PR helped you catch that the if statement was backwards before merging this into the rest of the code.

Reply:
As it turns out there were a lot of other logical errors. Don't know why I didn't catch them though, they were pretty obvious 😓. Anyway they should all be fixed now...